pages/login.page.js

```diff
@@ -0,0 +1,35 @@
+//import $ from 'webdriverio/build/commands/browser/$';
+import BasePage from '../pages/base.page';
+
+class LoginPage extends BasePage {
+
+   //WebElements
+   get usernameInput(){ return $('#username') }
+   get passwordInput(){return $('#password')}
+
+
+   //-------------------------------------------------------------------------------------------------------//
+
+   /**
+    * Write username and password and press Enter
+    * @param {String} username 
+    * @param {String} password
+    */
+   async enter(username, password) {
```

Comment:
This method basically encapsulates the action of logging in. `enter()` could work, but there's cases when you don't really *enter* the site (when login fails because of wrong credentials, etc). I think plain `logIn()` or `attemptLogin()` better describes what this method does. Using your terminology, I think even `attemptAccessingSite()` could be more descriptive as well. Consider renaming it.

Reply:
the name is because of the method it implements... enter instead of click

Reply:
Yeah, I'm just saying that is not descriptive enough, as it's not super explicit that is performing a log in action when you read it. My main argument against it is that it can happen that you call that method but you don't really *enter* the site's home page after it (like when it's called with invalid credentials), that makes it confusing. 

Prefer some more descriptive name, like the ones I suggested or some other that better expresses that you are attempting to log in to the site.

Reply:
done...